[PATCH v3]  example: ipsec_crypto: enhance runtime scripts

asasidharan opened the pull request:
This PR enhances the ipsec_crypto application runtime scripts similar to that being done for ipsec_api application in https://github.com/OpenDataPlane/odp/pull/1265.

For the ipsec_crypto application enabling all the run scripts by default. In the pktio_env script based on the IPSEC_APP_MODE, skipping the live / router mode tests in the CI.

MatiasElo reviewed (commented):
Couple minor comments but otherwise looks good.
- `pktio_env` is missing from .gitignore
- Rebase required

asasidharan commented on example/ipsec_crypto/odp_ipsec_crypto_run_ah_in.sh:
```diff
@@ -4,9 +4,40 @@
--s 192.168.222.2,192.168.111.2,loop2,loop1,10,100 \
--c 2 -m $1
+
+# IPSEC_APP_MODE: 0 - STAND ALONE, 1 - LIVE, 2 - ROUTER
```
Ack. Will fix this in v2.

asasidharan commented on example/ipsec_crypto/odp_ipsec_crypto_run_router.sh:
```diff
@@ -3,7 +3,55 @@
+PID=app_pid
+
+($STDBUF \
+ ./odp_ipsec_crypto -i $IF0,$IF2 \
+	-r 192.168.111.2/32,$IF0,$MAC_ADDR_IF1 \
+	-r 192.168.222.2/32,$IF2,$MAC_ADDR_IF3 \
```
Ack. Will fix this in v2.

asasidharan commented:
With v2:
- Rebased PR to latest master branch.
- Fixed typo for "STANDALONE" in all the runtime scripts.
- Fixed incorrect environment variables used in ROUTER mode script.

MatiasElo commented:
`pktio_env` is still missing from `example/ipsec_crypto/.gitignore`, but otherwise the PR looks good. You can add my reviewed-by tag while you fix that.

asasidharan commented:
> 
> 
> `pktio_env` is still missing from `example/ipsec_crypto/.gitignore`, but otherwise the PR looks good. You can add my reviewed-by tag while you fix that.

Sorry, missed to update .gitignore file. Updated this with v3. Also added your Reviewed-by tag.
Thank you @MatiasElo for reviewing https://github.com/OpenDataPlane/odp/pull/1265 as well as this PR.
